osbs/build/build_request.py

```diff
@@ -791,6 +832,9 @@ def render(self, validate=True):
         self.render_koji_promote(use_auth=use_auth)
         self.render_sendmail()
 
+
+        self.render_customizations()
```

Comment:
Shouldn't this be done at the beginning of `render`? Disabling and enabling a certain plugin in some cases requires other modifications.